src/components/balances/AssetsTable/index.tsx

```diff
@@ -185,7 +185,7 @@ const AssetsTable = ({
               sticky: true,
               collapsed: item.tokenInfo.address === hidingAsset,
               content: (
-                <Box display="flex" flexDirection="row" gap={1} alignItems="center">
+                <Stack direction="row" spacing={1} alignItems="center">
```

Comment:
Looks like it didn't work here for some reason it keeps using Box.
<img width="1266" alt="Screenshot 2024-12-09 at 16 27 58" src="https://github.com/user-attachments/assets/e0cc588b-39d6-44be-840a-ac91eb283a26">

Reply:
I noticed a couple other places. Will close the PR, not worth the trouble.